### PR TITLE
add config description for opentracing

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@tangenta 能否请一两位研发老师看看？谢谢~

**Review by zhongzc — APPROVED** [review]

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -482,6 +482,72 @@ prepare 语句的 plan cache 设置。
 + 最小值：0
 + 最大值：1
 
+## opentracing
+
```
Comment (TomShawn) [inline tidb-configuration-file.md:486]:
```suggestion

opentracing 的相关的设置。

```

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -482,6 +482,74 @@ prepare 语句的 plan cache 设置。
...
+opentracing 的相关的设置。
+
+### `enable`
+
++ 开启 opentracing 跟踪 TiDB 的部分组件调用开销。注意开启后会有一定的性能损失。
```
Comment (TomShawn) [inline tidb-configuration-file.md:491]: 比如哪些组件呢？
```suggestion
+ 开启 opentracing 跟踪 TiDB 部分组件的调用开销。注意开启后会有一定的性能损失。
```

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -482,6 +482,74 @@ prepare 语句的 plan cache 设置。
...
++ 默认值：false
+
+### `rpc-metrics`
+
++ 开启 rpc metrics。
++ 默认值：false
+
+## opentracing.sampler
+
```
Comment (TomShawn) [inline tidb-configuration-file.md:500]:
```suggestion

opentracing.sampler 相关的设置。

```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -482,6 +482,74 @@ prepare 语句的 plan cache 设置。
...
+### `type`
+
++ opentracing 采样器的类型。
++ 默认值："const"
++ 可选值：const, probabilistic, rateLimiting, remote。
```
Comment (TomShawn) [inline tidb-configuration-file.md:505]:
```suggestion
+ 可选值："const"，"probabilistic"，"rateLimiting"，remote"
```

File: tidb-configuration-file.md [hunk 5]
```diff
@@ -482,6 +482,74 @@ prepare 语句的 plan cache 设置。
...
++ 采样器可追踪的最大操作数。如果一个操作没有被追踪，会启用默认的 probabilistic 采样器。
++ 默认值：0
+
+### `sampling-refresh-interval`
+
++ 控制远程轮询 jaeger-agent 采样策略的频率。
++ 默认值：0
+
+## opentracing.reporter
+
```
Comment (TomShawn) [inline tidb-configuration-file.md:532]:
```suggestion

opentracing.reporter 相关的设置。

```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**